bedrock/mozorg/templates/mozorg/home/includes/fx100-promo.html

```diff
@@ -0,0 +1,17 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+<section class="fx100-hero">
+  <div class="mzp-l-content mzp-t-content-md">
+  <div class="fx100-hero-title-container">
+    <div class="fx100-hero-wordmark">Firefox 100</div>
+    <h1 class="fx100-hero-title">Building a better internet since 2004</h1>
+  </div>
+  <div class="fx100-hero-cta-container">
+    <a class="mzp-c-button mzp-t-product" href="{{ url('firefox.browsers.index') }}">Get the new Firefox</a>
```

Comment:
Just confirming this should link to the main /browsers page and not /new ?

Reply:
Yeah this is the URL that dan suggested